_data/simple-icons.json

```diff
@@ -1475,6 +1475,12 @@
             "source": "https://appwrite.io/assets",
             "guidelines": "https://appwrite.io/assets"
         },
+        {
+            "title": "Aptos",
+            "hex": "000000",
```

Comment:
```suggestion
            "hex": "5BFFFC",
```
Their accent color definitely isn't just black. I'm seeing `#5BFFFC` in some places, like the header logo on hover?

Reply:
I don't think so, if you check their social, the logo is always in black and white.

Reply:
I mean you can see the colored logo right in the favicon 😁 
![CleanShot 2024-10-10 at 15 39 48](https://github.com/user-attachments/assets/1a2136b7-6795-4e12-938e-52640d08ccff)

Reply:
?, it's black and white.
![Screenshot 2024-10-10 at 13 56 51](https://github.com/user-attachments/assets/81bd3a80-81e9-4264-9d2a-c8c4fd042959)
![Screenshot 2024-10-10 at 13 58 23](https://github.com/user-attachments/assets/23dd135f-c17a-4e2b-9357-12b9a98a1484)

Reply:
Go to https://media.aptosfoundation.org/1684136828-favicon.svg, which is the link used for the favicon in the document `<head>`. Has color!

Reply:
![Screenshot 2024-10-10 at 14 03 27](https://github.com/user-attachments/assets/c96f7e48-21b2-40a5-b1ec-e0764c235e1b)
![Screenshot 2024-10-10 at 14 02 31](https://github.com/user-attachments/assets/8166f6d6-fac6-4366-878d-175dd6210d47)
it's always use in black and white

Reply:
What browser are you using? 😅 
I'm seeing `#06F7F7` in the source of the SVG... What does the SVG contents look like for you?

![CleanShot 2024-10-10 at 16 08 07](https://github.com/user-attachments/assets/2d4709ad-9af8-4227-95b6-af597be06169)

Reply:
it's always used black and white, and people will expect black and white

![Screenshot 2024-10-10 at 14 07 47](https://github.com/user-attachments/assets/998eccc1-bc41-4786-a747-d8aa419db57b)
![Screenshot 2024-10-10 at 14 07 52](https://github.com/user-attachments/assets/5f4360d7-a4ef-4058-9ac4-74dbf719753e)
![Screenshot 2024-10-10 at 14 07 59](https://github.com/user-attachments/assets/b4f9e220-ba4b-428a-ace6-32c4b187d3e3)
![Screenshot 2024-10-10 at 14 08 03](https://github.com/user-attachments/assets/2c45d521-0d9d-4b52-aa94-c43b3a735a25)
![Screenshot 2024-10-10 at 14 08 18](https://github.com/user-attachments/assets/0a8100b5-ea43-4f40-9f3c-0710249fd5a4)

Reply:
Please reach out to them for confirmation then on the brand color.

Reply:
Here are the oficial symbols

![Screenshot 2024-10-10 at 14 17 28](https://github.com/user-attachments/assets/49439d46-a0e7-4645-87d6-b29166e5cad1)
![Aptos_mark_BLK](https://github.com/user-attachments/assets/50fc25ad-e9e8-4c9c-ae2c-613a820b8fd1)

![Aptos_mark_WHT](https://github.com/user-attachments/assets/e9d61b97-8df6-40ff-be37-dc9c0ed626c9)

Reply:
@uncenter I don't understand, why?, it's obvious what it is.

Reply:
We disagree, I think it is unclear, let's reach out! No harm in doing so. We have to wait for their permission on including the icon in the first place anyway.

Reply:
from the guidelines: "If no official color can be identified, use the brand's primary web color or the most prominent color in the logo itself (please indicate why you choose the particular color in your pull request)."

Reply:
I think the main colors are black and white. When someone use brand color from simple-icons, they expect to use the colors used by the brand. if the brand always use black and white that is what people will use. if not even the brand uses 5BFFFC in their socials, people will not be expecting that color. It's common sense. But well , I will ask anyway ... sigh.

Reply:
The brand does not always use black and white, as demonstrated by the multiple areas of the website using the cyan accent and with the favicon being colored. If you disagree, that is fine - but you must reach out to the brand to confirm, because it is clearly ambiguous as we cannot come to agreement on this. The "common sense" you mention would be to reach out for clarification!